src/components/login/login.js

```diff
@@ -187,8 +188,15 @@ class Login extends React.Component {
                   maxInputsLength={24}
                   onFill={this.checkPassphrase}
                 />
-              </div>
 
+                <DiscreetModeToggle
+                  className={styles.discreetMode}
+                  isEnable={settings.discreetMode}
+                  onClick={() => settingsUpdated({ discreetMode: !settings.discreetMode })}
```

Comment:
IMO `onClick` and `isEnable` should be only inside the `DiscreetModeToggle` component, because no matter where we use the toggle, this behavior should always be the same and we would be duplicating the code in all pages that use the toggle. This implies `DiscreetModeToggle` needs `HOC` similar to https://github.com/LiskHQ/lisk-hub/blob/dc58d3862d30e2c1b9be4c3e49d5478a6563cd47/src/components/discreetMode/index.js